packer/epoch.json

```diff
@@ -52,7 +52,7 @@
     },
     {
       "type": "ansible",
-      "user": "master",
+      "user": "ubuntu",
```

Comment:
I notice this ubuntu is different from the master set in other script - unclear to me full picture. I guess this is something specific to packer and the ami used?

Reply:
The difference is because AWS and OS are using different base images with different sudo users

Reply:
We unify this during the image builds